src/KafkaFlow.Admin/ITelemetryCache.cs

```diff
@@ -0,0 +1,27 @@
+namespace KafkaFlow.Admin
+{
+    using System.Collections.Generic;
+    using KafkaFlow.Admin.Messages;
+
+    /// <summary>
+    /// Used to implement a cache provider to manage telemetry data
+    /// </summary>
+    public interface ITelemetryCache
```

Comment:
I think we can change the name to ITelemetryStore(age?)

Reply:
done